plugins/Deflect/DeflectPlugin.cpp

```diff
@@ -81,31 +81,32 @@ class DeflectPlugin::Impl
                 _closeStream();
         }
 
-        if (_previousHost != _params.getHostname())
-        {
-            _params.setEnabled(true);
-            _previousHost = _params.getHostname();
-        }
-
         const bool deflectEnabled = _params.getEnabled();
         if (_stream && _stream->isConnected() && !deflectEnabled)
```

Comment:
This statement is not needed with the change suggested above as the stream would have been closed in the first if statement.